docs/source/references/isi.md

```diff
@@ -0,0 +1,14 @@
+# Iroha Special Instructions
+
+Because Iroha Special Instructions is a very important topic among Iroha functionality we made
+this reference with detailed description of every aspect related to **ISI**.
+
+## TL;DR
+
+In Iroha 2.0 we have Iroha Special Instructions (inspired by commands from Iroha 1 and smart contracts in other systems).
+They are provided out of the box for all legal actions and contain implementations with Permissions check inside. 
+But we have several Instructions for composition like ‘If’, ‘Sequence’, etc.
+So using them Iroha modules and dependent applications can build their own custom commands - which are built
```

Comment:
`So by using the Iroha modules, dependent applications can build.. ` something like that?

Reply:
I think this sentence is still not corrected.

Reply:
Idea is that complex ISI can be used by Iroha modules and other applications to build their custom instructions. Can you help with text for that?

Reply:
`Therefore Iroha modules and other applications are able to compose their custom complex commands on top of OOB ISI with the help of these composition oriented instructions.` Maybe something like that?